### PR TITLE
New post: livre blanc : l'architecture logicielle, tout ce que vous

### PR DESCRIPTION
**Pull request by eko**
devez savoir

### CODE REVIEW

**Comment by ElevenWilson** [comment]
4 words might be misspelled, please check them: lang, téléchargement, Télécharger, vcomposieux

**Comment by ElevenWilson** [comment]
4 words might be misspelled, please check them: lang, téléchargement, Télécharger, vcomposieux

File: _posts/fr/2018-04-05-livre-blanc-architecture.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Livre blanc : L'architecture logicielle, tout ce que vous devez savoir"
+lang: fr
+permalink: /fr/livre-blanc-architecture-logicielle-tout-ce-que-vous-devez-savoir/
+excerpt: "Avec l'arrivée des différentes technologies permettant de développer des applications web ainsi que des différents frameworks associés, outils et évolution des standards du web et de développement, le besoin de définir une architecture claire sur les projets est rapidement apparue."
```
Comment (ch3ric) [inline _posts/fr/2018-04-05-livre-blanc-architecture.md]: apparu

File: _posts/fr/2018-04-05-livre-blanc-architecture.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - livre-blanc
+    - architecture
+cover: /assets/2018-04-05-livre-blanc-architecture/cover.jpg
+---
+
+Premier de sa lignée, Eleven Labs a aujourd'hui le plaisir de vous offrir ce livre blanc (gratuit) sur l'architecture logicielle en téléchargement.
+
+# Présentation
+
+Avec l'arrivée des différentes technologies permettant de développer des applications web ainsi que des différents frameworks associés, outils et évolution des standards du web et de développement, le besoin de définir une architecture claire sur les projets est rapidement apparue.
```
Comment (ch3ric) [inline _posts/fr/2018-04-05-livre-blanc-architecture.md]: apparu

File: _posts/fr/2018-04-05-livre-blanc-architecture.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+# À qui s'adresse ce livre ?
+
+Ce livre s'adresse aussi bien :
+* À des décisionnaires afin de les aider à mieux comprendre ce qu'un architecte peut leur apporter, ce qu'ils sont en droit d'attendre d'un architecte,
+* À des développeurs afin de les éclaircir sur le rôle d'un architecte au sein de leur projet ainsi que de comprendre dans quels cas certains patterns peuvent être appliqués et ce qu'ils engendrent,
+* Enfin, à toute personne dans le domaine du web (chefs de projets, product owners, ...) : cela vous permettra en effet d'éclaircir votre vision de la technique et de vous faire un allier qui vous aidera à répondre au mieux à votre besoin et à construire une application plus adaptée.
+
+[![Télécharger livre blanc]({{site.baseurl}}/assets/2018-04-05-livre-blanc-architecture/button.jpg)](https://s3.eu-west-3.amazonaws.com/files.eleven-labs.com/ebooks/){: .center-image }
```
Comment (ch3ric) [inline _posts/fr/2018-04-05-livre-blanc-architecture.md]: - I'm pretty sure we will want to know how many people downloaded this ebook (at least, and maybe from where): are the S3 analytics enough for that? or do we need to track this link with google analytics?
- note: for now this link is not accessible: private bucket on S3: do not forget to change access rules on S3 when this PR will be merged

File: _posts/fr/2018-04-05-livre-blanc-architecture.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Après plusieurs expériences sur ce type de poste, ce métier étant parfois mal compris par certains clients, il m'a semblé intéressant de préparer un petit document afin de décrire de façon large, ce qu'est l'architecture logicielle mais aussi quels peuvent être les différents rôles d'un architecte au cours des différents projets sur lesquels il est amené à travailler.
+
+# À qui s'adresse ce livre ?
+
+Ce livre s'adresse aussi bien :
+* À des décisionnaires afin de les aider à mieux comprendre ce qu'un architecte peut leur apporter, ce qu'ils sont en droit d'attendre d'un architecte,
+* À des développeurs afin de les éclaircir sur le rôle d'un architecte au sein de leur projet ainsi que de comprendre dans quels cas certains patterns peuvent être appliqués et ce qu'ils engendrent,
+* Enfin, à toute personne dans le domaine du web (chefs de projets, product owners, ...) : cela vous permettra en effet d'éclaircir votre vision de la technique et de vous faire un allier qui vous aidera à répondre au mieux à votre besoin et à construire une application plus adaptée.
```
Comment (ch3ric) [inline _posts/fr/2018-04-05-livre-blanc-architecture.md]: répétition "éclaircir": clarifier ?

**Comment by ElevenWilson** [comment]
4 words might be misspelled, please check them: lang, téléchargement, Télécharger, vcomposieux

**Comment by ElevenWilson** [comment]
5 words might be misspelled, please check them: Codelabs, lang, qu'architecte, Télécharger, vcomposieux

**Comment by ElevenWilson** [comment]
5 words might be misspelled, please check them: Codelabs, lang, qu'architecte, Télécharger, vcomposieux

**Comment by ElevenWilson** [comment]
5 words might be misspelled, please check them: Codelabs, lang, qu'architecte, Télécharger, vcomposieux

**Comment by ElevenWilson** [comment]
5 words might be misspelled, please check them: Codelabs, lang, qu'architecte, Télécharger, vcomposieux

**Pull request merged**